src/codegen/class.ts

```diff
@@ -112,7 +112,7 @@ export async function* generateClasses(
   yield `import { LanguageTag, parseLanguageTag }
     from "@phensley/language-tag";\n`;
   yield `import { decode as decodeMultibase, encode as encodeMultibase }
-    from "multibase";`;
+    from "../runtime/multibase/index.ts";`;
```

Comment:
```suggestion
    from "${runtimePath}/multibase/index.ts";`;
```

To avoid hard-coding paths.